### ISSUE TITLE
Write-up of database structure

### ISSUE THREAD

**Issue by dave-mills** [stats4sd/SSD-Resources-Demo#114]
So we have a proper map for future reference.

**Comment by chrismclarke** [comment]
Agreed, I've started to put together a google doc for site documentation. Share link sent or comment-only link (for anyone interested) below:
https://docs.google.com/document/d/13OABObUNk6dqXC6xUX8y9MYJ7Ut15b8uu3ElqQ8gNF4/edit?usp=sharing

**Comment by dave-mills** [comment]
Rough database structure added to your document. There are a few bits that will probably change as we remove legacy keys and update things.
